CONTRIBUTING.md

```diff
@@ -100,9 +100,17 @@ Code
  - 80 character line length
  - commas last
  - declare variables in the outermost scope that they are used
+ - camelCase for variables in JavaScript
+ - snake_case for variables in Python
 
 In general, try to make your code blend in with the surrounding code.
 
+## Pull Requests
+ 
+ - link back to the original issue(s) whenever possible
+ - commits should be squashed before merging
+ - large pull requests should be broken into multiple logically cohesive commits (or separate pull requests) if possible
```

Comment:
I'd rather say `broken into separate pull requests (or multiple logically cohesive commits)`, since we'd rather have multiple pull requests.